src/qtt/measurements/scans.py

```diff
@@ -234,6 +234,10 @@ def get_instrument(instr, station=None):
     if isinstance(instr, Instrument):
         return instr
 
+    if isinstance(instr, AcquisitionScopeInterface):
+        instr.name = 'scope_reader'
```

Comment:
Why does the `get_instrument` method update the name of the `AcquisitionScopeInterface` object?

Reply:
Because of line 1762 in qtt\measurements\scans.py:
```
is_simulation = minstrument.name.startswith('sdigitizer')
```

Reply:
@CumulonimbusCalvus Better then replace that line with 
```
is_simulation = isinstance(minstrument, SimulationDigitizer)
```
(or similar)

Changing names of instruments is not something we want. And why was this an issue? The AcquisitionScopeInterface did not have name `sdigitizer` right?

Reply:
The AcquisionScopeInterface does indeed not have a name. Therefore, `minstrument.name` raises an error. I do like your idea (did not know that the simulated digitizer was of type SimulationDigitizer): ```is_simulation = isinstance(minstrument, SimulationDigitizer)```